### ISSUE TITLE
Thnks

### ISSUE THREAD

**Issue by witaliy12** [pmd/pmd#4748]
**Affects PMD Version:** 

**Rule:**

Please provide the rule name and a link to the rule documentation:
<https://docs.pmd-code.org/latest/pmd_rules_XXX_XXX.html#XXX>

**Description:**

**Code Sample demonstrating the issue:**

```java

```

**Expected outcome:**

PMD reports a violation at line ..., but that's wrong. That's a false positive.

**Running PMD through:** *[CLI | Ant | Maven | Gradle | Designer | Other]*
